analyze.py

```diff
@@ -70,7 +69,7 @@ def is_fubar(results):
             has_wrong_pubkey = True
             logging.debug(conn['pubkey'][0] + ' is a fubar pubkey size')
             fubar = True
-        if ec_kex and pubkey_bits < min_ec_size:
+        if ec_kex and pubkey_bits < 160:
```

Comment:
Hm, that contradicts the JSON versions:

https://statics.tls.security.mozilla.org/server-side-tls-conf-4.0.json 
https://statics.tls.security.mozilla.org/server-side-tls-conf.json

which still list 160 for old. Not sure which is correct here, and I don't have a MozillaWiki account to leave a comment on the discussion page.